cmd/serve.go

```diff
@@ -69,18 +79,42 @@ func cmdServe(cmd *cobra.Command, args []string) {
 		case <-ctx.Done():
 			return
 		case e := <-events:
-			switch e {
+			switch e.Type {
 			case kube.NewPodEvent:
-				metrics.GlobalPodMetrics.TotalNumberPods.Add(1)
+				podlog := log.New("pod", e.PodSelfLink)
+
+				cgPath := buildCgroupPath(cgroupRoot, e.PodUID, e.PodQOSClass)
```

Comment:
is it possible to get this from the Kubernetes API rather than assuming the path is built in a specific way?

/cc @asymmetric @nhlfr